src/observers/link_click_observer.ts

```diff
@@ -60,9 +60,10 @@ export class LinkClickObserver {
     )
   }
 
-  findLinkFromClickTarget(target: EventTarget | null) {
+  findLinkFromClickTarget(target: EventTarget | null): HTMLAnchorElement | undefined {
```

Comment:
@izaguirrejoe Explicitly specifying the return type removed the need of having to declare the `else { ... }` statement. This type used to be `HTMLAnchorElement | null | undefined` and I don't think changing that to `HTMLAnchorElement | undefined` will cause any issues.

Reply:
I agree, that looks simpler! At first I felt uncomfortable returning undefined, but that's what the original function did anyways, and it make the PR a one-liner.